components/org.wso2.carbon.identity.application.authenticator.oidc/pom.xml

```diff
@@ -20,7 +20,7 @@
         <groupId>org.wso2.carbon.identity.outbound.auth.oidc</groupId>
         <artifactId>identity-application-auth-oidc</artifactId>
         <relativePath>../../pom.xml</relativePath>
-        <version>5.5.2</version>
+        <version>5.5.10</version>
```

Comment:
update to 5.5.10-SNAPSHOT

Reply:
Resolved with https://github.com/wso2-extensions/identity-outbound-auth-oidc/pull/86/commits/746f3872e911b0271060e0e39d0b50d214633e50.